x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -332,79 +336,90 @@ export class Explorer extends React.Component {
             />
             <EuiSpacer size="m" />
 
-            {annotationsData.length > 0 && (
-              <>
+            <EuiAccordion
```

Comment:
@mdefazio do you think it's worth putting an `EuiPanel` around the Annotations content? I think having the content in a panel looks cleaner, but the argument against is that it draws attention away from the Anomalies content, which arguably is more important information.

<img width="1789" alt="Screen Shot 2020-07-01 at 11 21 22 AM" src="https://user-images.githubusercontent.com/7405507/86269026-0bd7dd80-bbc1-11ea-9a46-9790c033d539.png">

Reply:
100% agree that it could use a panel around it.  (And didn't know where to comment for this in relation to this PR).  

My response to your argument against it drawing attention away would be that if the anomalies panel is more important, do we simply switch the order?

Reply:
Let's add the panel. If we moved annotations below the anomalies section, I'd be concerned the user would miss it altogether.